sdk/textanalytics/ai-text-analytics/samples/javascript/analyzeSentimentWithOpinionMining.js

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how to analyze sentiment on a more granular level,
+ * mining individual opinions from reviews (also known as aspect-based 
+ * sentiment analysis). This feature is only available for clients with api 
+ * version v3.1-preview.1.
+ * 
+ * In this sample, we will be a customer who is trying to figure out whether 
```

Comment:
not sure the description accomplishes what the sample is actually doing. I will say if you are going to follow the sample we have in Python and .NET then better execute it completely with the analysis of the aspects and it's opinions.

Reply:
description has been updated to better match what is in the sample.